src/utils/StreamHighlightUtils.ts

```diff
@@ -97,6 +93,10 @@ export function getRestHighlightsForAllTerms(
       .sortBy('length')
       .value();
 
+    if (uniqueTermsToHighlight.length === 0) {
+      return;
```

Comment:
Should we return an empty array here since the return signature is `IHighlight[]`?

Reply:
The return statement is in an anonymous function, in the _.each. It does not return the whole function, it just stops the loop.